DOL.WHD.Section14c.Web/src/modules/components/sectionWageData/wageDataPayTypeFormDirective.test.js

```diff
@@ -0,0 +1,21 @@
+describe('wageDataPayTypeForm', function() {
+
+    beforeEach(module('14c'));
+
+    var element, rootScope;
+    beforeEach(function () {
+        element = angular.element('<wage-data-pay-type-form/>');
+    	inject(function ($rootScope, $compile, _$q_, responsesService) {
+            $q = _$q_;
+            rootScope = $rootScope;
+            responses = $q.defer();
+            mockResponsesService = responsesService;
+            spyOn(mockResponsesService, 'getQuestionResponses').and.returnValue(responses.promise);
+            $compile(element)(rootScope);
+        });
+    });
+
+    it('invoke directive', function() {
+        rootScope.$digest();
```

Comment:
updated directive unit tests to assert it was created https://github.com/AppliedIS/dol-whd-14c/commit/e73d8d8cc8291dbbcf47a7da435a7ad963cc54a4